odl/discr/grid.py

```diff
@@ -681,9 +684,46 @@ def append(self, other):
         See Also
         --------
         insert
+        extend
         """
         return self.insert(self.ndim, other)
 
+    def extend(self, seq):
+        """Return a copy of ``self``, extended by all members of ``seq``.
+
+        Parameters
+        ----------
+        seq : sequence of `RectGrid`
```

Comment:
There really is no proper way out, right? An option would be to simply let append take several arguments.

```
def append(self, *stuff):
    If len(stuff) > 1:
        return self.append(stuff[0]).append(stuff [1:])
    else:
        #current code
```

Would be an implementation